realtime/server.js

```diff
@@ -1,14 +1,16 @@
-var io = require('socket.io');
-var req = require('request');
-var fs = require('fs');
-var winston = require('winston');
+var io = require('socket.io'); //include websockets
+var req = require('request'); //include request(supports http)
+var fs = require('fs'); //include file system
+var winston = require('winston'); //include winston(multi-transport async logging library)
```

Comment:
The multi-transport and async properties of winston are not something we use and do not concern us.